yarn.lock

```diff
@@ -25259,6 +25295,18 @@ terser-webpack-plugin@^3.0.0:
     terser "^4.8.0"
     webpack-sources "^1.4.3"
 
+terser-webpack-plugin@^5.1.3:
```

Comment:
Looks like you need to update our specified version of `terser-webpack-plugin` at the root to 5.1.3 as well.

Reply:
I deduped `terser`, `terser-webpack-plugin`, `acorn` & `enhanced-resolve` 👍